### PR TITLE
fix: deploying to ECS - last part

### LINKED ISSUE #124: As a DevOps, I want to run the OpenTelemetry Stack and the application on ECS.
Currently, you can deploy and run the application locally. However, in AWS when ECS is deployed with Terraform, there are some fixes that need to be implemented:

- OpenTelemetry collector service needs to load the file `collector-config.yam` in the task definition. Currently is pushing the image from the official public repo, but since we need to configure how to receive, process, and export data in the `config.yaml` file, this specific file needs to be loaded in the task definition as well.

- The RDS database is being created. However, the tables that are generated locally from the file `docker_postgres_init.sql `seem that are not created in RDS. So this needs to be automated as well.

- The backend only has the DB password set, but it is missing the rest of the info to connect to the RDS database;

- The load balancer is not reaching the frontend when is deployed in ECS. The security rules seem to be fine, and it is a problem in the frontend app when it's deployed into ECS;

- The backend and frontend images are not being generated when it is deployed the Terraform code. It is necessary to automate the build and push the image to ECR (probably using workflows); #131 

- The backend is failing because there is no healthcheck (and ECS requires one)

- The services can't talk to each other the way the repo is currently set up when deploying to ECS

- The backend is restarting because it can't connect to the collector

- Frontend needs fixing, it won't build correctly. #167 
  - Uncaught ReferenceError: require is not defined at index.0e2e872d.js:117:52841
  - `require("lodash.merge")`, for some reason (browsers don't support `require`)

### PR DESCRIPTION
**Pull request by Pehesi97**
Closes #124.

Steps to test it:

```
terragrunt run-all apply

docker buildx build --platform=linux/amd64 --build-arg APP_VERSION=1.0.0 -t <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-be-img-repo:latest -f Backend.dockerfile .

docker buildx build --platform=linux/amd64 --build-arg API_URL=<ALB URL>:3000 --build-arg OTLP_COLLECTOR_URL=<ALB URL>:8083/v1/traces -t <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-fe-img-repo:latest -f Frontend.dockerfile .

docker buildx build --platform=linux/amd64 -t 101259067028.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-collector-img-repo:latest -f Collector.dockerfile .

aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com && docker push <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-be-img-repo:latest

docker push <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-fe-img-repo:latest

docker push <AWS account ID>.dkr.ecr.us-east-1.amazonaws.com/dev-otlp-collector-img-repo:latest
```

### CODE REVIEW

**Comment by Pehesi97** [comment]
I think we should merge this PR and create 2 issues:

- frontend is failing on ECS because of a `require` line (browsers don't support `require`)
  - Uncaught ReferenceError: require is not defined at index.0e2e872d.js:117:52841
  - `require("lodash.merge")` - I've looked it up, it's not a dependency - probably a subdependency

- the images for the frontend, backend and the collector should be built on a workflow

File: collector-config.yaml [hunk 1]
```diff
@@ -8,12 +8,14 @@ receivers:
           allowed_origins:
             - http://*
             - https://*
+            - "*"
```
Comment (kurtmilam) [inline collector-config.yaml:11]: @Pehesi97 Why do we need to allow origins other than `http://*` and `https://*` (which already seem pretty permissive)?

**Review by kurtmilam — COMMENTED** [review]
I have one question about the wildcard in the collector `allowed_origins` config.

File: collector-config.yaml (hunk 1 above)
Comment (Pehesi97) [reply collector-config.yaml:11]: Honestly, I added that line for testing only. I was having problems on connecting the frontend and backend to the collector, and this was an attempt to debug.
It should be OK to remove that line.

Comment (jdarling) [reply collector-config.yaml:11]: We will clean this up with #186 #187 and #188

**Review by jdarling — APPROVED** [review]

**Pull request merged**